test/fixture/assert.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
We need to change this to `AVA Snapshot v1`. We don't want to mislead people with being fully compatible with Jest and it's also confusing for AVA users that it doesn't say AVA.

We might need to open an issue on Jest about getting an option for this, but for now I guess we can just read the file in, modify it, and write it out again.